lib/emailnotifications_queue_handler.ts

```diff
@@ -186,14 +215,22 @@ export async function handleNotification(
     ? messageContent.subject
     : "Un nuovo avviso per te.";
 
-  // converts the markdown body to HTML
-  // TODO: handle errors / validate the markdown
-  const bodyHtml = (await markdownToHtml.process(
+  const documentHtmlOrError = await generateDocumentHtml(
+    subject,
     messageContent.bodyMarkdown
-  )).toString();
+  );
+
+  if (documentHtmlOrError.isLeft) {
```

Comment:
as a broader issue, we should remember to consider that with all this message processing there are a lot of possible failure reasons (a lot more than when sending plain text). 

the parser(s) must be resilient to errors or we risk to miss a lot of targets.

Reply:
indeed, that's why we log errors and we will monitor the system closely

Reply:
indeed, that's why we log errors and we will monitor the system closely

Reply:
indeed, that's why we log errors and we will monitor the system closely